extensions/wikia/GlobalNavigation/scripts/GlobalNavigationSearch.js

```diff
@@ -20,6 +21,15 @@ require(['jquery', 'wikia.browserDetect', 'GlobalNavigationiOSScrollFix'], funct
 		} else {
 			setPropertiesOnInput(true);
 		}
+
+		//Disables search button until the user has entered at least one character.
+		$searchInput.keyup(function () {
+			if ($searchInput.val()) {
+				$searchSubmit.removeAttr('disabled');
```

Comment:
[Boolean attributes](http://www.w3.org/TR/html4/intro/sgmltut.html#h-3.3.4.2) don't have `true`/`false` value, in their full form they go like `disabled="disabled"`.